.github/workflows/ci.yml

```diff
@@ -2,26 +2,22 @@ name: CI
 
 on:
   push:
-    branches:
-      - master
-  pull_request:
-    branches:
-      - master
 
 jobs:
   build:
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
-        node-version: [12.x, 14.x]
+        node-version: [18, 20, 22]
```

Comment:
It might help to drop a link to https://github.com/actions/runner-images#support-policy as comment before.